arcade/paths.py

```diff
@@ -18,16 +28,30 @@
 def _spot_is_blocked(position: Point,
                      moving_sprite: Sprite,
                      blocking_sprites: SpriteList) -> bool:
+    """
+    Return if position is blocked
+
+    Returns whether or not the moving_sprite is blocked
+    at position is blocked of the sprite in blocking_sprites
+
+    Args:
+        position: position to put moving_sprite at
+        moving_sprite: Sprite to use
+        blocking_sprites: List of Sprites to check against
+
+    Returns:
+        If the Sprite would hit anything in blocking_sprites at the position
+    """
     original_pos = moving_sprite.position
     moving_sprite.position = position
     hit_list = check_for_collision_with_list(moving_sprite, blocking_sprites)
     moving_sprite.position = original_pos
     return len(hit_list) > 0
 
 
-def _heuristic(start: Point, goal: Point):
+def _heuristic(start: Point, goal: Point) -> float:
     """
-    Returns the heuristic of the 2 poitns
+    Returns the heuristic of the 2 points
```

Comment:
```suggestion
    Returns a heuristic value for the passed points.
```
The original docstring may be subtly incorrect, as may the method body. I would handle the docstring as above for now.